src/main/kotlin/Handler.kt

```diff
@@ -27,22 +27,24 @@ class Handler : RequestHandler<Request, ByteArray?> {
     private val tableNameUtil = TableNameUtil()
 
     override fun handleRequest(input: Request, context: Context?): ByteArray? {
-        logger.info("Input received for topic ${input.topic} with key ${input.key} and timestamp ${input.timestamp} for delete request ${input.deleteRequest}")
-        val topic = input.topic.toByteArray()
+        logger.info("Input received for topic '${input.topic}' with key '${input.key}' and timestamp '${input.timestamp}' for delete request '${input.deleteRequest}' and table per topic setting of '${input.useTablePerTopic}'")
         val timestamp = input.timestamp
         val isDeleteRequest = input.deleteRequest
         val useTablePerTopic = input.useTablePerTopic
-        val family = HbaseConfig.dataFamily.toByteArray()
+        val family = (if (useTablePerTopic) "cf" else HbaseConfig.dataFamily).toByteArray()
```

Comment:
what's `cf`?

Reply:
The column family name used in HTME and HDI with the new paradigm.

Reply:
so `cf` = `column family`?

Reply:
Yep